sdk/storage/Azure.Storage.Common/swagger/Generator/src/generator.ts

```diff
@@ -1109,7 +1109,7 @@ function generateSerialize(w: IndentWriter, service: IService, type: IObjectType
         const properties = <IProperty[]>Object.values(type.properties);
         for (const property of properties) {
             let current = elementName;
-            const { xname: childName } = getXmlShape(property.name, { ...type.xml, name: property.name });
+            const { xname: childName } = getXmlShape(property.name, property.xml);
```

Comment:
This scares me, but I guess there were no other differences in the generated code?

Reply:
Correct